app/helpers/records_helper.rb

```diff
@@ -160,6 +181,24 @@ def more_tab_count(tab, categories)
   end
 
   def facet_name(name)
-    name == 'decade' ? 'Date' : name.capitalize.gsub('_', ' ')
+    case name 
+    when 'decade'
+      'Date'
+    when 'primary_collection'
+      'Collection'
+    else
+      name.titleize
+    end
   end  
+
+  def facet_list(facet)
```

Comment:
This will need to be updated to take `name` and `values` from the changes here https://github.com/DigitalNZ/supplejack_website/pull/22/files#r58474545